### PR TITLE
update to forge v1.4.0 from master after release.

### PR DESCRIPTION
**Pull request by gonuke**
Hoping I did this correctly...

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks Paul!  Comments in-line.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0rc1" %}
+{% set version = "1.4.0" %}
 {% set org = "cyclus" %}
-{% set branch = "develop" %}
+{% set branch = "master" %}
 {% set sha256 = "7a6bd1334f3fd13ddac73fbb7a5218b958b793b81bda26552e9c3abbf255a905" %}
```
Comment (scopatz) [inline recipe/meta.yaml]: The SHA256 sum of the tarball has probably changed.

**Comment by scopatz** [comment]
Also, the other 3 open PRs need to go in now too.... I can try to take care of this if you'd rather not futz with it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -65,3 +65,7 @@ about:
 extra:
   recipe-maintainers:
     - scopatz
+    - gonuke
+    - baaaaam
```
Comment (scopatz) [inline recipe/meta.yaml]: I am not sure, but this may need to have the correct casing.

Comment (gonuke) [inline recipe/meta.yaml]: OK - both cases work in the URL?? - but who knows how it will work here

**Comment by jakirkham** [comment]
Needs to be [re-rendered]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) with `conda-smithy` version `1.3.3`.

File: recipe/meta.yaml (hunk 1 above)
Comment (gonuke) [inline recipe/meta.yaml]: I don't think of tarballs having SHAs.  Do you mean the commit SHA for the version that generates the tarball?  I guess I'm not sure how this works...  I can't seem to find documentation either.

**Comment by gonuke** [comment]
I'm happy for you to do it... was just trying to learn :smile:

**Comment by gonuke** [comment]
This may be minutes/seconds premature as well...

**Comment by scopatz** [comment]
> I'm happy for you to do it... was just trying to learn 😄

Thanks! I just expect this particular case to be a little frustrating because of the backlog, and wanted to warn you so that the first couple of experiences weren't terrible. :) 

> Needs to be re-rendered with conda-smithy version 1.3.3.

got it, will do @jakirkham 

> This may be minutes/seconds premature as well...

OK! I'll wait until the signal is given and then merge all of PRs into a single branch, rerender and resubmit.

File: recipe/meta.yaml (hunk 2 above)
Comment (scopatz) [inline recipe/meta.yaml]: Yeah, conda-forge can tends to be more precise, so I don't know how it will work out.

File: recipe/meta.yaml (hunk 1 above)
Comment (scopatz) [inline recipe/meta.yaml]: So this is actually a `sha256sum` of the tarball, like you'd do with md5.  You have to download the tarball and run the sha256sum command against it.  

@conda-forge/core are the people to notify about doc issues, in general.  Though in this case the docs are in conda-build http://conda.pydata.org/docs/building/meta-yaml.html#source-from-tarball

Comment (gonuke) [inline recipe/meta.yaml]: Which SHA should this be exactly?

Comment (scopatz) [inline recipe/meta.yaml]: It is the hash of the whole tarball. This value is then checked by conda-build to makes sure that the tarball that conda-build downloads is correct (no download corruption, man-in-the-middle, etc).  The hash algorithm that is recommended by conda/conda-forge is the SHA256 algo. Though we could use this to MD5 or SHA1 if we wanted.

It is not the commit SHA, which is the hash of only that changeset in git.  This hash needs to be generated independently of the version control system. Does this make sense?

**Comment by gonuke** [comment]
I'm not sure which SHA I should be providing - is there documentation somewhere?  Or will you take care of that, too?

**Comment by scopatz** [comment]
> I'm not sure which SHA I should be providing - is there documentation somewhere? Or will you take care of that, too?

Yeah, I'll take care of it, but for future regerence you need to do something like the following:

```console
$ wget https://github.com/cyclus/cyclus/archive/master.tar.gz
$ sha256sum master.tar.gz
```

**Comment by Baaaaam** [comment]
Cyclus Cycamore should be ready.

**Comment by scopatz** [comment]
Hey @jakirkham - when you said conda-smithy v1.3.3, did you mean v1.3.3+ or that I should downgrade from v1.4.2?

**Pull request merged**